### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by vianamp** [jacobgil/keras-grad-cam#11]
Hi. Good job putting this together. I was wondering whether `loss=K.sum(...)` should be replaced with `K.max(...)` once we are interested on the output for the class with highest output value only. What do you think?

**Comment by jacobgil** [comment]
Notice that the last layer type is target_category_loss,  
K.sum here is the same as K.max, only a single class is activated.

**Comment by vianamp** [comment]
Absolutely. Thanks.
